### DISCUSSION TITLE
🌟 Starstruck Exchange 🌟

### DISCUSSION THREAD

**Discussion by xLevitate** [Schweinepriester/github-profile-achievements#435] [Q&A] [answered]
https://github.com/xLevitate/neptune

leave a star on my repo and reply with your repo and I will star it back!

**Comment by esbatudaisuki** [comment]
https://github.com/aditrachman/rental-pacar
can i get star back

**Reply by Jesulayomy** [reply]
Think its private so can't see it

**Reply by abhinit21** [reply]
Yeah! @esbatudaisuki 
Can't access.

**Reply by ArielMAJ** [reply]
Please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Buedenbender** [comment]


**Reply by abhinit21** [reply]
Done 👍@Buedenbender @Jesulayomy 

Can't I get back too : https://github.com/abhinit21/abhinit21

**Reply by Preetiraj3697** [reply]
Done Can i get back too https://github.com/Preetiraj3697/Sheela_Ecommerce_Website and https://github.com/Preetiraj3697/30dayofJavascript

**Reply by bensonarafat** [reply]
done https://github.com/codepymdev/grad

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by kensoz** [comment]


**Reply by Jesulayomy** [reply]
I did
https://github.com/Jesulayomy/calculator

**Reply by Preetiraj3697** [reply]
Done Can i get back too https://github.com/Preetiraj3697/Sheela_Ecommerce_Website and https://github.com/Preetiraj3697/30dayofJavascript

**Reply by bensonarafat** [reply]
done https://github.com/codepymdev/grad

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by montasim** [comment, marked as answer]
@xLevitate can you please select this as an answer?

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Jesulayomy** [comment]
Done
https://github.com/Jesulayomy/calculator

**Reply by montasim** [reply]
Done @Jesulayomy

**Reply by Jesulayomy** [reply]
Thank you

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Mateo-Wallace** [comment]


**Reply by Jesulayomy** [reply]
Gave you a star too;
https://github.com/Jesulayomy/calculator

**Reply by montasim** [reply]
Done @Mateo-Wallace

**Reply by Preetiraj3697** [reply]
Done Can i get back too https://github.com/Preetiraj3697/Sheela_Ecommerce_Website and https://github.com/Preetiraj3697/30dayofJavascript

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by abhinit21** [comment]
Done 👍
https://github.com/abhinit21/abhinit21

**Reply by Jesulayomy** [reply]
Also done
https://github.com/Jesulayomy/calculator

**Reply by montasim** [reply]
Done @abhinit21 
https://github.com/montasim/montasim

**Reply by kensoz** [reply]
@abhinit21  Done, would you star me back :)
https://github.com/kensoz/FEC

**Reply by Preetiraj3697** [reply]
@abhinit21 Done Can i get back too https://github.com/Preetiraj3697/Sheela_Ecommerce_Website and https://github.com/Preetiraj3697/30dayofJavascript

**Reply by abhinit21** [reply]
Done 👍 great Repo btw

**Reply by itspasindu** [reply]
please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by abhisheks008** [comment]
Star this repo, https://github.com/abhisheks008/DL-Simplified

**Reply by Jesulayomy** [reply]
https://github.com/Jesulayomy/calculator
Done ty

**Reply by Jesulayomy** [reply]
Nice repo :v also Twitter image doesn't load in the readme for some reason

**Reply by abhisheks008** [reply]
Updated that! ✅

**Reply by Preetiraj3697** [reply]
@abhisheks008 Done Can i get back too https://github.com/Preetiraj3697/Sheela_Ecommerce_Website and https://github.com/Preetiraj3697/30dayofJavascript

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by 1RB** [comment]
star this repo please 
https://github.com/1RB/Authworria

**Reply by 1RB** [reply]
done btw

**Reply by 1RB** [reply]
congrats @xLevitate i was the 16th star

**Reply by abhinit21** [reply]
Done 👍
https://github.com/abhinit21/abhinit21

**Reply by Jesulayomy** [reply]
Done too
https://github.com/Jesulayomy/calculator

**Reply by 1RB** [reply]
> Done 👍 https://github.com/abhinit21/abhinit21

thank you and done

**Reply by 1RB** [reply]
> Done too https://github.com/Jesulayomy/calculator

done

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Preetiraj3697** [comment]
Star this repo https://github.com/Preetiraj3697/Sheela_Ecommerce_Website and https://github.com/Preetiraj3697/30dayofJavascript you leave a star on my repo and reply with your repo and I will star it back!

**Reply by Jesulayomy** [reply]
Gotcha so I'm giving two back xd
https://github.com/Jesulayomy/calculator
https://github.com/Jesulayomy/tic-tac-toe-C

**Reply by Preetiraj3697** [reply]
Done

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
> Gotcha so I'm giving two back xd https://github.com/Jesulayomy/calculator https://github.com/Jesulayomy/tic-tac-toe-C

please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Preetiraj3697** [comment]
Done @xLevitate can i get star back  https://github.com/Preetiraj3697/Sheela_Ecommerce_Website and https://github.com/Preetiraj3697/30dayofJavascript

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by Preetiraj3697** [reply]
Done

**Reply by ArielMAJ** [reply]
Ty!

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by muhammadnurulahsan** [comment]
star and ill star your repo back [link it below]
https://github.com/muhammadnurulahsan/Carpenco-Store-Client-Side
https://github.com/muhammadnurulahsan/Carpenco-Store-Server-Side

**Reply by Jesulayomy** [reply]
Done
https://github.com/Jesulayomy/calculator
https://github.com/Jesulayomy/tic-tac-toe-C

**Reply by muhammadnurulahsan** [reply]
done

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@Jesulayomy @muhammadnurulahsan I still haven't received your star, so I'll be removing yours. Lmk when you star me so I'll star you back again!

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by ryker-uptycs** [comment]
Done

**Comment by bstummer** [comment]
Star and I'll star your repo back
https://github.com/bstummer/openskill.lua

@xLevitate

**Reply by Jesulayomy** [reply]
Gave you one
https://github.com/Jesulayomy/calculator

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by DushanSenadheera** [comment]
Hello! can anyone help me to achieve starstruck achievement by giving some starts to this repo https://github.com/DushanSenadheera/Algorithms-and-Data-Structures

**Reply by Jesulayomy** [reply]
Done https://github.com/Jesulayomy/calculator

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by ArielMAJ** [comment]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@xLevitate I still haven't received your star, so I'll be removing yours. Lmk when you star me so I'll star you back again!

**Reply by Jesulayomy** [reply]
@ArielMAJ done

**Reply by ArielMAJ** [reply]
@Jesulayomy tyvm! I starred you back

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by bensonarafat** [comment]
https://github.com/codepymdev/grad

**Reply by HasanSafarli** [reply]
starred you! please star my repo too:

https://github.com/HasanSafarli/food-delivery-app

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by itspasindu** [comment]
done .
please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by ArielMAJ** [reply]
@itspasindu I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
> @itspasindu I starred you, please star my repo too: https://github.com/ArielMAJ/test.me Feel free to contribute as well, there are many somewhat "simple" issues.

done

**Reply by ArielMAJ** [reply]
Ty

**Comment by itspasindu** [comment]
> https://github.com/xLevitate/neptune
> 
> leave a star on my repo and reply with your repo and I will star it back!

please star my repo too:
https://github.com/itspasindu/isipathana-college-demo-site

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by treeplate** [comment]
treeplate/syllad

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Reply by xommmax** [reply]
Done! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Comment by Rantoryu** [comment]
@xLevitate 
Done, please star my repo: https://github.com/Rantoryu/Personal_Website
I will star back anyone in the comment to this!
